### ISSUE TITLE
How is the effect on the test set?

### ISSUE THREAD

**Issue by zbaoye** [Justin-Tan/generative-compression#22]
Hi, author:
    When I re-train this model, it works pretty well on training dataset after 250 epoch. but it doesn't appear to work very well on the testing dataset. Is your result the same?

Thank you!

**Comment by zhouqingyi616** [comment]
Same. There's no validation evaluating step. Need help.

**Comment by ChenWei007HB** [comment]
I met the same problem, when retraining the model, the test result of Kodak datasets can not surpass BPG method, even if tested on the model provided by the author.

**Comment by imsleepy711** [comment]
Hi, I met the same problem. Do you have some solution now? Thank you!
